packages/platform/atoms/destination-calendar/DestinationReminderSelector.tsx

```diff
@@ -0,0 +1,49 @@
+"use client";
+
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { Select } from "@calcom/ui/components/form";
+import type { ReminderMinutes } from "@calcom/trpc/server/routers/viewer/calendars/setDestinationReminder.schema";
+
+const REMINDER_OPTIONS: Array<{ value: ReminderMinutes; label: string }> = [
+  { value: 0, label: "just_in_time" },
+  { value: 10, label: "remind_minutes_before" },
+  { value: 30, label: "remind_minutes_before" },
+  { value: 60, label: "remind_minutes_before" },
+];
+
+interface DestinationReminderSelectorProps {
+  value: ReminderMinutes;
+  onChange: (value: ReminderMinutes) => void;
+  isPending?: boolean;
+}
+
+export const DestinationReminderSelector = ({
+  value,
+  onChange,
+  isPending,
+}: DestinationReminderSelectorProps) => {
+  const { t } = useLocale();
+
+  const options = REMINDER_OPTIONS.map((opt) => ({
+    value: opt.value,
+    label: opt.label === "just_in_time" ? t(opt.label) : t(opt.label, { count: opt.value }),
```

Comment:
I think you can directly use t() without optional chaining... if count is empty, it'd still choose opt.label